x-pack/plugins/security_solution/public/management/cypress/cypress_serverless_qa.config.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { defineCypressConfig } from '@kbn/cypress-config';
+import { getCypressBaseConfig } from './cypress_base.config';
+
+// eslint-disable-next-line import/no-default-export
+export default defineCypressConfig(
+  getCypressBaseConfig({
+    e2e: {
+      experimentalCspAllowList: ['default-src', 'script-src', 'script-src-elem'],
+    },
+    env: {
+      // Uncomment to enable logging
+      // TOOLING_LOG_LEVEL: 'verbose',
+      grepTags: '@serverlessQA --@skipInServerless',
```

Comment:
We are adding the `@skipInServerlessMKI`, label as well,  check: https://github.com/elastic/kibana/pull/181371

Reply:
If you want to be align with us, in this case the `@serverlessQA` should not be here and be `@serverless` since we want to have all the tests to be executed as part of the periodic pipeline and just a subset of them to be executed as part of the kibana release process.

Reply:
I see. So, insted of including tests to be run in this periodic pipeline we will be excluding from the pool of all serverless tests, correct?

Reply:
Not exactly, the labels are going to work as follow:

`@serverless:` The test is executed as part of the PR check process and in the periodic pipeline (MKI environment).

`@serverlessQA`: The test is executed as part of the kibana release process in the QA environment (MKI environment).

`@skipInServerless:` The test is skipped from being executed in CI as part of the PR check and is skipped from being executed in any MKI environment.

`@skipInServerlessMKI: The test is skipped from being executed in any MKI environment but it will continue being executed as part of the PR process if the @serverless tag is present.`

So by default all the tests marked as `@serverless` are going to be executed as part of the periodic pipeline but not as part of the release process. In case you want to promote a test to be part of the release pipeline, you have to explicitely add the `@serverlessQA` if not the test is going to be excluded even containing the `@serverless` label.